.github/workflows/build-beta.yml

```diff
@@ -16,17 +26,11 @@ jobs:
           # However, we need the head commit (the latest commit pushed to the source branch)
           # because in the workflow, we would like to parse the latest commit message.
           # Specifying `ref` ensures that the head commit is checked out directly.
-          # For a `pull_request` event, the head commit hash is `github.event.pull_request.head.sha`.
-          # For a `push` event, the head commit hash is `github.sha`.
-          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+          ref: ${{ env.HEAD_COMMIT_HASH }}
 
       - name: Needs beta build
-        # For a `pull_request` event, the branch is `github.head_ref``.
-        # For a `push` event, the branch is `github.ref_name`.
-        if: ${{ (github.head_ref || github.ref_name) != 'master' }}
+        if: ${{ env.BRANCH != 'master' }}
```

Comment:
Should this have `&& !env.IS_FORK`?

A fork would never trigger a beta.

Or maybe the whole workflow/job should get skipped on a fork?